### PR TITLE
Issue170 fix

### PR DESCRIPTION
**Pull request by HansChaudry**
Fixed failing workflow issue with R-CMD-Check.

### CODE REVIEW

File: .github/workflows/check-standard.yaml [hunk 1]
```diff
@@ -1,15 +1,16 @@
 # For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
 # https://community.rstudio.com/new-topic?category=Package%20development&tags=github-actions
+name: R-CMD-check
+
 on:
   push:
     branches:
       - main
+      - issue170_fix
```
Comment (chisingh) [inline .github/workflows/check-standard.yaml:9]: Let's remove this line before merging. The main branch would still run this workflow.

File: .github/workflows/check-standard.yaml [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
   pull_request:
     branches:
       - main
 
-name: R-CMD-check
```
Comment (chisingh) [inline .github/workflows/check-standard.yaml:11]: Plese restore this deleted line.

**Comment by chisingh** [comment]
> [CLA assistant check](https://cla.opensource.microsoft.com/microsoft/datamations?pullRequest=177) Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/datamations?pullRequest=177) before we can accept your contribution.❌ HansChaudry [sign now](https://cla.opensource.microsoft.com/microsoft/datamations?pullRequest=177)You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/microsoft/datamations?pullRequest=177) it.

@HansChaudry, you need to sign the CLA once.

File: .github/workflows/check-standard.yaml (hunk 1 above)
Comment (HansChaudry) [reply .github/workflows/check-standard.yaml:9]: > Just want clarification. Are you referring to the line with the name of my branch?

**Review by chisingh — APPROVED** [review]

**Review by LinqingZ — APPROVED** [review]

**Review by LinqingZ — APPROVED** [review]

**Review by GeorgeIskander25 — APPROVED** [review]

**Pull request merged**